traversability_estimation:
  ros__parameters:
    # Footprint parameters
    footprint:
      footprint_polygon: [0.0, 0.0, 0.5, 0.0, 0.5, 0.5, 0.0, 0.50]  # Vertices of footprint polygon in form of x1, y1, x2, y2, ...
      circular_footprint_radius: 0.541
      circular_footprint_radius_inscribed: 0.30
      circular_footprint_offset: 0.15
      footprint_frame_id: base_link
      traversability_default: 0.3
      verify_roughness_footprint: false
      check_robot_inclination: false
